problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,52 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  while (this.stack1.length !== 0) {
```

Comment:
```suggestion
  while (this.stack1.length) {
```
위와 같이 수정할 수 있을 것 같아요.